src/screens/points/content/PointsContent.tsx

```diff
@@ -0,0 +1,796 @@
+      icon = '🥇';
+      break;
+    case 2:
+      gradient = ['#FBFCFE', '#B3BCC7'];
+      icon = '🥈';
+      break;
+    case 3:
+      gradient = ['#DE8F38', '#AE5F25'];
+      icon = '🥉';
+      break;
+    case 4:
+      icon = '􀁀';
+      break;
+    case 5:
+      icon = '􀁂';
+      break;
+    case 6:
+      icon = '􀁄';
+      break;
+    case 7:
+      icon = '􀁆';
+      break;
+    case 8:
+      icon = '􀁈';
+      break;
+    case 9:
+      icon = '􀁊';
+      break;
+    case 10:
+      icon = '􀓵';
+      break;
+    default:
+      icon = '􀁜';
+      break;
+  }
+
+  const formattedPoints = points.toLocaleString('en-US');
+
+  return (
+    <ButtonPressAnimation
+      onPress={() =>
+        navigate(Routes.PROFILE_SHEET, {
+          address: ens ?? address,
+          fromRoute: 'PointsScreen',
+        })
+      }
+    >
+      <Box
+        paddingVertical="10px"
+        flexDirection="row"
+        justifyContent="space-between"
+        alignItems="center"
+      >
+        <Inline space="10px" alignVertical="center">
+          {avatarURL ? (
+            <Box
+              as={Image}
+              source={{ uri: maybeSignUri(avatarURL) }}
+              style={{ width: 36, height: 36 }}
+              borderRadius={18}
+              background="surfaceSecondaryElevated"
+              shadow="12px"
+            />
+          ) : (
+            <AccentColorProvider
+              color={
+                colors.avatarBackgrounds[addressHashedColorIndex(address) ?? 0]
+              }
+            >
+              <Box
+                style={{
+                  width: 36,
+                  height: 36,
+                }}
+                background="accent"
+                borderRadius={18}
+                shadow="12px"
+                alignItems="center"
+                justifyContent="center"
+              >
+                <Text
+                  align="center"
+                  weight="bold"
+                  size="20pt"
+                  color="labelTertiary"
+                  containsEmoji
+                >
+                  {addressHashedEmoji(address) ?? '􀉪'}
+                </Text>
+              </Box>
+            </AccentColorProvider>
+          )}
+          <Stack space="8px">
+            <Box style={{ maxWidth: 145 }}>
+              <Text
+                color="label"
+                weight="bold"
+                size="15pt"
+                ellipsizeMode="middle"
+                numberOfLines={1}
+              >
+                {ens ? ens : formatAddress(address, 4, 5)}
+              </Text>
+            </Box>
+            {STREAKS_ENABLED && (
+              <Inline space="2px" alignVertical="center">
+                <Text color="labelQuaternary" size="11pt" weight="bold">
+                  􀙬
+                </Text>
+                <Text color="labelQuaternary" size="13pt" weight="semibold">
+                  {`40 ${i18n.t(i18n.l.points.points.days)}`}
```

Comment:
prob will want to change this hardcoded value if were using a remote flag

Reply:
removed this in follow up pr that i'm working on